src/components/studies/participants/ParticipantManager.tsx

```diff
@@ -729,38 +782,40 @@ const ParticipantManager: FunctionComponent<ParticipantManagerProps> = () => {
                         </Button>
                       )}
 
-                      <ParticipantDownloadTrigger
-                        onDownload={() =>
-                          downloadParticipants(isAllSelected ? 'ALL' : 'SOME')
-                        }
-                        fileDownloadUrl={fileDownloadUrl}
-                        hasItems={
-                          !!data?.items?.length &&
-                          selectedParticipantIds[tab].length > 0
-                        }
-                        onDone={() => {
-                          URL.revokeObjectURL(fileDownloadUrl!)
-                          setFileDownloadUrl(undefined)
-                        }}>
-                        <>
-                          <img
-                            src={DownloadIcon}
-                            style={{
-                              marginRight: '6px',
-                              opacity:
-                                selectedParticipantIds[tab].length === 0
-                                  ? 0.5
-                                  : 1,
-                            }}></img>
-                          {!loadingIndicators.isDownloading ? (
-                            'StudyParticipants.csv'
-                          ) : (
-                            <CircularProgress size={24} />
-                          )}
-                        </>
-                      </ParticipantDownloadTrigger>
-
-                      {tab !== 'WITHDRAWN' && (
+                      {!displayNoParticipantsPage && (
```

Comment:
I don't think we should have those repeated checks. We should replace the entire area with the 'no participants' screen. Also, let's have it in a separate file.